index.d.ts

```diff
@@ -8,6 +8,7 @@ declare module '@primer/components' {
     as?: React.ReactType
     className?: string
     css?: string
+    sx?: object
```

Comment:
Can we use the `SystemStyleObject` type from `@styled-system/css` so we get more type hinting for the `sx` prop?

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/styled-system__css/index.d.ts#L401